Tests/UnitTests/Core/EventData/FreeTrackParametersTests.cpp

```diff
@@ -20,13 +20,20 @@ namespace Test {
 /// @brief Unit test for free parameters
 ///
 BOOST_AUTO_TEST_CASE(free_initialization) {
-  Vector3D pos(0., 1., 2.);
-  double t = 3.;
+  Vector4D pos4(0., 1., 2., 4.);
   Vector3D dir(4., 5., 6.);
-  double qop = 7.;
+  double p = 7.;
+  double q = 1.;
 
   FreeVector params;
-  params << pos.x(), pos.y(), pos.z(), t, dir.x(), dir.y(), dir.z(), qop;
```

Comment:
I understand why you do this, but it's painful.

Reply:
I do not disagree, but it is the only way if we want to allow configurable parameter ordering.